src/firebolt_db/firebolt_async_dialect.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import annotations
```

Comment:
nit: firebolt_async_dialect.py -> async_dialect.py, the parent folder already says firebolt

Reply:
Same could also apply to firebolt_dialect.py -> dialect.py

Reply:
Fair point. Need to check if there's any usage, as this would be a non-backwards-compatible change.

Reply:
Filed a ticket on removing the need to hardcode the path. We can rename those files as a part of it and probably release a major version. https://packboard.atlassian.net/browse/FIR-10025